vmdb/app/controllers/chargeback_controller.rb

```diff
@@ -147,6 +147,7 @@ def cb_rate_edit
       @sb[:rate] = @edit[:rate] if @edit && @edit[:rate]
       if @edit[:new][:description].nil? || @edit[:new][:description] == ""
         add_flash(_("%s is required") % "Description", :error)
+        @in_a_form = true
```

Comment:
@GregP when there is an error we shouldn't do replace_right_cell, instead we should just replace flash message div, we might not need @in_a_form = true and call to replace_right_cell in this case, try replacing those with
render :update do |page|
            page.replace("flash_msg_div", :partial => "layouts/flash_msg")
          end